File: Miscellaneous_Hashes.cs
using System;
using System.Numerics;
using System.Collections.Generic;

namespace Miscellaneous_CS
{
    class Program
    {
        static void Main(string[] args)
        {
            char[] message = "merry xmas".ToCharArray();

            Hashes one_way = new Hashes();
            String hashed = one_way.hashWord(message);

            Console.WriteLine("Message is '" + String.Join("", message) +
                "';\nMessage hash is " + hashed);
        }
    }
}

File: Miscellaneous_Hashes.java
package miscellaneous;

public class Miscellaneous {

    public static void main(String[] args) {
        char[] message = "merry xmas".toCharArray();
        
        Hashes one_way = new Hashes();
        String hashed = one_way.hashWord(message);
        
        System.out.println("Message is '" + String.valueOf(message) +
                "';\nMessage hash is " + hashed);
    }

}